ui/src/app/pages/admin/user/admin-user-bypass.tsx

```diff
@@ -54,6 +55,44 @@ const getBypassedModules = (user: AdminTableUser): Array<AccessModule> => {
   ];
 };
 
+const moduleToToggleProps: Record<
```

Comment:
Thanks for teaching me about `Record`. Very useful.

Reply:
One downside is that `Record` doesn't validate that every enum value has a mapping so adding a new access module won't break the typing.